db/schema/patch-007-008.sql

```diff
@@ -0,0 +1,175 @@
+-- Add a lockedAt column for account locking.
+
+ALTER TABLE accounts ADD COLUMN lockedAt BIGINT UNSIGNED DEFAULT NULL;
+
+-- the codes to unlock an account
+CREATE TABLE IF NOT EXISTS accountUnlockCodes (
+  uid BINARY(16) PRIMARY KEY NOT NULL,
+  unlockCode BINARY(16) NOT NULL,
+  FOREIGN KEY (uid)
+      REFERENCES accounts(uid)
+      ON DELETE CASCADE
```

Comment:
This stands out to me because we haven't been doing explicit `FOREIGN KEY` clauses, but rather just cleaning up all the linked tables by hand.  See e.g. the `keyFetchTokens` table declaration.

It does seem like a good idea though!  I've no idea if there are any performance concerns we need to be aware of, but I imagine it'll be fine.

I have a slight concern about confusing potential future maintainers if we have more-than-one-way-to-do-it depending on what table you're working with.